ui/widgets/sidebar/src/components/sidebar-component.tsx

```diff
@@ -1,61 +1,81 @@
-import Button from '@akashaorg/design-system-core/lib/components/Button';
-import ListSidebarApps from './list-sidebar-apps';
-import BasicCardBox from '@akashaorg/design-system-core/lib/components/BasicCardBox';
-import { startMobileSidebarHidingBreakpoint } from '@akashaorg/design-system-core/lib/utils/breakpoints';
 import {
   getProfileImageVersionsWithMediaUrl,
   useGetLogin,
   useLogout,
   LOGIN_STATE_KEY,
+  useDismissedCard,
 } from '@akashaorg/ui-awf-hooks';
-import { useQueryClient } from '@tanstack/react-query';
 import getSDK from '@akashaorg/awf-sdk';
-import { AUTH_EVENTS, WEB3_EVENTS } from '@akashaorg/typings/sdk/events';
 
-const SidebarComponent: React.FC<RootComponentProps> = props => {
-  const [isMobile, setIsMobile] = React.useState(
-    window.matchMedia(startMobileSidebarHidingBreakpoint).matches,
-  );
+import Anchor from '@akashaorg/design-system-core/lib/components/Anchor';
+import Avatar from '@akashaorg/design-system-core/lib/components/Avatar';
+import BasicCardBox from '@akashaorg/design-system-core/lib/components/BasicCardBox';
+import Box from '@akashaorg/design-system-core/lib/components/Box';
+import Button from '@akashaorg/design-system-core/lib/components/Button';
+import DidField from '@akashaorg/design-system-core/lib/components/DidField';
+import Stack from '@akashaorg/design-system-core/lib/components/Stack';
+import Text from '@akashaorg/design-system-core/lib/components/Text';
+import TextLine from '@akashaorg/design-system-core/lib/components/TextLine';
+import { startMobileSidebarHidingBreakpoint } from '@akashaorg/design-system-core/lib/utils/breakpoints';
 
-  React.useEffect(() => {
-    const mql = window.matchMedia(startMobileSidebarHidingBreakpoint);
-    const resize = () => {
-      setIsMobile(mql.matches);
-    };
-    window.addEventListener('resize', resize);
-    return () => {
-      window.removeEventListener('resize', resize);
-    };
-  }, []);
+import ListSidebarApps from './list-sidebar-apps';
+import SidebarCTACard from './cta-card';
 
+const SidebarComponent: React.FC<RootComponentProps> = props => {
   const {
     uiEvents,
     plugins,
     worldConfig: { defaultApps, socialLinks },
   } = props;
+
+  const [isMobile, setIsMobile] = useState(
+    window.matchMedia(startMobileSidebarHidingBreakpoint).matches,
+  );
   const [routeData, setRouteData] = useState(null);
   const [activeOption, setActiveOption] = useState<IMenuItem | null>(null);
-  const [isLoading, setIsLoading] = React.useState(false);
-  const [profileName, setProfileName] = React.useState('Guest');
+  const [isLoading, setIsLoading] = useState(false);
+  const [profileName, setProfileName] = useState('Guest');
+
+  const subtitleUi = useRef(null);
+
   const { t } = useTranslation('ui-widget-sidebar');
 
   const loginQuery = useGetLogin();
   const logoutQuery = useLogout();
   const queryClient = useQueryClient();
+  const [dismissed, setDismissed] = useDismissedCard();
```

Comment:
Imho the `useDismissedCard` hook should make use of a `Set` instead of `Array`. You can't dismiss multiple times the same card id, it also saves you from a lot of pain if for example a card is registered multiple times.